### ISSUE TITLE
API error: Invalid format. data.body.message.body is missing and it is required.

### ISSUE THREAD

**Issue by daithimaolain** [RoushTech/RollbarDotNet#45]
When calling SendException() in my middleware:

var response = await _rollbar.SendException(ex);

I get the following exception:

```StatusCode: 422, ReasonPhrase: 'Unprocessable Entity', Version: 1.1, Content: System.Net.Http.StreamContent, Headers:
{
  Connection: keep-alive
  Date: Fri, 10 Mar 2017 09:44:05 GMT
  Server: nginx
  Access-Control-Allow-Origin: *
  Access-Control-Allow-Credentials: true
  X-Response-Time: 6ms
  Content-Length: 100
  Content-Type: application/json; charset=utf-8
}
```

The associated error in rollbar is:

API error: Invalid format. data.body.message.body is missing and it is required.

As far as I can tell my configuration looks okay, the Rollbar class appears to be properly instantiated with 6 builders and a single exception builder. I get the same error if i use services.AddRollbar(), services.AddRollbarWeb() or use your own middleware class: app.UseRollbarExceptionHandler(); 

SendMessage function works as expected.

**Comment by daithimaolain** [comment]
I tracked the problem down to this: API error: Invalid format. data.body must have either a 'trace', 'trace_chain', 'message' or 'crash_report' property, (but not more than one).

As a temporary workaround:

`public class CustomExceptionBuilder : IExceptionBuilder
    {
        public void Execute(Payload payload, System.Exception exception)
        {
            payload.Data.Body.Message = null;
        }
    }`

then:

`services.AddRollbarWeb();
 services.AddSingleton<IExceptionBuilder, CustomExceptionBuilder>();`

**Comment by StrangeWill** [comment]
Yep, slipped through due to a known hole in our testing. Went ahead and patched that hole for 0.5.1 and will see about getting a release later today.
